script/README.md

```diff
@@ -85,3 +87,11 @@ This script is used to determine whether the students successfully fixed their g
 This script is used to reset the github-games repository in githubschool using the template in githubtraining. It will also delete the copy of the game from the githubteacher account.
 
 **Usage:** `script/reset-game`
+
+## Troubleshoot
+
+  1. `jq not found`
+  If you are on Windows and download the .exe (instead of using Chocolatey), you will get a file named `jq-win64` or similar. You would need to add the folder where you store the `.exe` to the PATH and rename the `.exe` to just `jq`.
```

Comment:
```suggestion
  1. `jq not found`
  
     If you are on Windows and download the .exe (instead of using Chocolatey), you will get a file named `jq-win64` or similar. You would need to add the folder where you store the `.exe` to the PATH and rename the `.exe` to just `jq`.
```